packages/web-components/fast-element/docs/api-report.md

```diff
@@ -880,6 +880,11 @@ export function volatile(target: {}, name: string | Accessor, descriptor: Proper
 // @public
 export function when<TSource = any, TReturn = any>(binding: Binding<TSource, TReturn>, templateOrTemplateBinding: SyntheticViewTemplate | Binding<TSource, SyntheticViewTemplate>): CaptureType<TSource>;
 
+// Warnings were encountered during analysis:
+//
+// dist/dts/components/fast-element.d.ts:73:5 - (ae-forgotten-export) The symbol "define" needs to be exported by the entry point index.d.ts
```

Comment:
Seems like these need to be exported. Is it not understanding the function overloads correctly?

Reply:
Yeah, I think so. I don't think it really matters in the end though.